assessment-api/open-api-specification.yml

```diff
@@ -99,10 +142,24 @@ components:
           items:
             $ref: "#/components/schemas/assessmentDetail"
     assessmentDetail:
+      allOf:
+        - $ref: "#/components/schemas/baseAssessmentDetail"
+        - type: 'object'
+          properties:
+            id:
```

Comment:
Seems strange that id is in the extended assessmentDetail and not in the base object

Reply:
You don't want to be able to amend the id as part of the patch